### ISSUE TITLE
retrieving all edges within a bounding box

### ISSUE THREAD

**Issue by kungfu303** [valhalla/valhalla#3439]
Is there a way to get all edges within a bounding box of coordinates?

I know that the /locate endpoint gets you all edges where certain points lie on, but not within a range of coordinates like a bbox.

Thanks!

**Comment by kevinkreiser** [comment]
locate lets you get all edges within a radius, so not a box but a circle

**Comment by kungfu303** [comment]
That would also work for us -- thanks!

**Comment by kungfu303** [comment]
How do you specify how long the radius is? Is there an option to do so?

**Comment by nilsnolde** [comment]
https://github.com/valhalla/valhalla/blob/master/docs/api/turn-by-turn/api-reference.md#locations ;)

**Comment by kungfu303** [comment]
Following up on this, 

I used /locate with the radius location parameter but I'm facing a situation where there seems to be a radius cap of 200 meters from my center point. 

For the following json input to locate, I tried varying the radius between 200 meters, 400 meters, 1000 meters, and 20000 meters. All of these outputted the same number of edges and the same output.

`{"verbose":true,"locations":[{"lat":37.78642650987315,"lon":-122.40641681884112, "radius": <I varied this>}], "costing":"bicycle","costing_options":{"bicycle":{}},"directions_options":{"units":"miles"}}`

How can I get the edges that lie greater than 200 meters from this center location?
